tests/test_update.py

```
@@ -483,23 +484,32 @@ async def test_async_install_reboots_when_needed(monkeypatch, make_config_entry,
     class FakeClient:
         def __init__(self):
             # planned sequence: first 'running', then 'done'
+            """Initialize FakeClient."""
             self._status_calls = [
                 {"status": "running"},
                 {"status": "done"},
             ]
             self.rebooted = False
 
         async def upgrade_firmware(self, upgrade_type):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `upgrade_firmware`

(ANN202)